config/prow/plugins.yaml

```diff
@@ -259,6 +260,19 @@ heart:
   - mirandachrist
   commentregexp: ".*"
 
+branch_cleaner:
+    # PreservedBranches is a map of org/repo branches
+    # format:
+    # ```
+    # preserved_branches:
+    # <org>: ["master", "release"]
+    # <org/repo>: ["master", "release"]
+    # ```
+    # branches in this allow map would be exempt from branch gc
+    # even if the branches are already merged into the target branch
+    preserved_branches:
+        "kubernetes-sigs/cloud-provider-azure": ["master", "gh-pages", "release-1.1", "release-1.0", "release-1.24", "release-0.7", "release-0.5", "release-0.1"]
```

Comment:
I think you missed `release-1.23` and `release-0.6`

Reply:
updated.